crates/bevy_ui/src/widget/image.rs

```diff
@@ -73,20 +70,20 @@ impl Measure for ImageMeasure {
     }
 }
 
+/// The filter type for `update_image_content_size_system` and `update_atlas_content_size_system`.
+#[cfg(feature = "bevy_text")]
+pub type UpdateImageFilter = (With<Node>, Without<bevy_text::Text>);
+/// The filter type for `update_image_content_size_system` and `update_atlas_content_size_system`.
+#[cfg(not(feature = "bevy_text"))]
+pub type UpdateImageFilter = With<Node>;
```

Comment:
```suggestion
/// The filter type for `update_image_content_size_system` and `update_atlas_content_size_system`.
#[cfg(feature = "bevy_text")]
type UpdateImageFilter = (With<Node>, Without<bevy_text::Text>);
/// The filter type for `update_image_content_size_system` and `update_atlas_content_size_system`.
#[cfg(not(feature = "bevy_text"))]
type UpdateImageFilter = With<Node>;
```

There's no need for these to be pub I think?

Reply:
Oooh, yeah looks like it. Nice. I thought it needed to be pub due to being a parameter of a pub functions. Let me update it.